src/plugins/cosmos/components/modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -19,86 +18,78 @@ export const useFormSend = () => {
     state: { wallet },
   } = useWallet()
 
-  type CosmosSdkChainFees = FeeData<KnownChainIds.CosmosMainnet> &
-    FeeData<KnownChainIds.OsmosisMainnet>
-
   const handleSend = async (data: SendInput) => {
-    if (wallet) {
-      try {
-        const adapter = chainAdapterManager.get(data.asset.chainId) as unknown as ChainAdapter<
-          KnownChainIds.CosmosMainnet | KnownChainIds.OsmosisMainnet
-        >
-        if (!adapter) throw new Error(`No adapter available for chainId ${data.asset.chainId}`)
-        const value = bnOrZero(data.cryptoAmount)
-          .times(bn(10).exponentiatedBy(data.asset.precision))
-          .toFixed(0)
+    if (!wallet) return
```

Comment:
re organized with early returns/throws to improve readability and reduce indent depth